spark-plugins/docs/DecisionTreeTrainer-sparksink.md

```diff
@@ -24,7 +24,10 @@ fields cannot be specified.
 featureFieldsToInclude and featureFieldsToExclude are empty, all fields except the label will be used for training. Both
  *featureFieldsToInclude* and *featureFieldsToExclude* fields cannot be specified.
 
-**cardinalityMapping:** Mapping of the feature to the cardinality of that feature; required for categorical features.
+**arityMapping:** List of the categorical features along with its arity value, where arity indicates the number
```

Comment:
cardinality is `a measure of the "number of elements of the set". `, whereas arity is the number of operands that a function takes. so IMO its should be cardinality. The description should be something simple like
"List of the categorical features along with  maximum number of unique values that feature can exist in. e.g daysOfTheWeek : 7"